CODEOWNERS

```diff
@@ -339,13 +339,9 @@
 
 # Tests
 /tests/                                   @nrfconnect/ncs-co-verification @katgiadla
-/tests/benchmarks/current_consumption/    @nrfconnect/ncs-low-level-test
-/tests/benchmarks/i2c_endless/            @nrfconnect/ncs-low-level-test
-/tests/benchmarks/multicore/              @carlescufi
```

Comment:
I think this 
"tests/benchmarks/multicore/              @carlescufi"
is still needed. Or are you fine with being removed @carlescufi from here?

Reply:
Let's wait for Carles response.

Reply:
I will restore that ownership.